src/librustc/ty/context.rs

```diff
@@ -2395,18 +2325,17 @@ pub fn keep_local<'tcx, T: ty::TypeFoldable<'tcx>>(x: &T) -> bool {
 }
 
 direct_interners!('tcx,
-    region: mk_region(|r: &RegionKind| r.keep_in_local_tcx()) -> RegionKind,
-    goal: mk_goal(|c: &GoalKind<'_>| keep_local(c)) -> GoalKind<'tcx>,
-    const_: mk_const(|c: &Const<'_>| keep_local(&c)) -> Const<'tcx>
+    region: mk_region -> RegionKind,
+    goal: mk_goal -> GoalKind<'tcx>,
+    const_: mk_const -> Const<'tcx>
```

Comment:
Maybe this should be changed to use the syntax `region: mk_region(RegionKind),`

Reply:
Done.